test/integration/render/scenarios/belongs/nin/scenario.js

```diff
@@ -12,5 +12,6 @@ const layer = new carto.Layer('layer', source, viz);
 
 layer.addTo(map);
 layer.on('loaded', () => {
+    console.log('wad');
```

Comment:
'wad' stands for...?

Reply:
hahaha, stands for `wadus`, I forgot to remove it